### DISCUSSION TITLE
How to know is supabase is down ?

### DISCUSSION THREAD

**Discussion by Melliti** [supabase/supabase#20928] [Questions] [answered]
Hello, 

I'd like to know if there is a way to know if Supabase is down or is facing any issue. I have a software tunning on supabase since months now and this morning (europe) it's down. I checked on X if people were facing the same issue and I found only one message by @rphlmr. So it's not general but I'm not alone either.

I have a production and a staging DB but both are unreachable.

Edit: I just found 3 additional post on X saying they are facing issue too.

Thanks

**Comment by rphlmr** [comment, marked as answer]
Hello 👋

You can report your issue here https://supabase.com/dashboard/support/new

I hope it is nothing too hard to solve :D

And to now when it's down: https://status.supabase.com/
